### PR TITLE
Add r-caper recipe

### PR DESCRIPTION
**Pull request by nick-youngblut**

### CODE REVIEW

**Comment by nick-youngblut** [comment]
Whoops, I might have gotten ahead of myself here. Is it OK to have >1 recipe per PR?

**Comment by mariusvniekerk** [comment]
Yeah, its fine

**Comment by nick-youngblut** [comment]
r-picante is failing for appveyor due to gcc missing:

```
C:/bld/r-picante_1526731100594/_h_env/R/../Library/mingw-w64/bin/gcc  -I"C:/bld/r-picante_1526731100594/_h_env/R/include" -DNDEBUG     -I"C:/bld/r-picante_1526731100594/_h_env/R/../Library/mingw-w64/include"     -O2 -Wall  -std=gnu99 -march=x86-64 -mtune=generic -c picante.c -o picante.o
/bin/sh: C:/bld/r-picante_1526731100594/_h_env/R/../Library/mingw-w64/bin/gcc: No such file or directory
C:/bld/r-picante_1526731100594/_h_env/R/etc/x64/Makeconf:208: recipe for target 'picante.o' failed
make: *** [picante.o] Error 127
```

I don't have much experience with windows. Should I modify the `bld.bat` file as shown in ["Building on Windows" docs](https://conda-forge.org/docs/buildwin.html)? I checked some other recipes like `r-lmer` and `r-rcpp` and they don't change the `bld.bat`, but I thought they would (maybe gcc isn't a problem).

File: recipes/r-caper/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+build:
+  number: 0
+  skip: true  # [win32]
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
```
Comment (mariusvniekerk) [inline recipes/r-caper/meta.yaml]: Try adding 

build:
  - {{ compiler('c') }}

File: recipes/r-picante/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+source:
+  fn: picante_{{ version }}.tar.gz
+  url:
+    - {{ cran_mirror }}/src/contrib/picante_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/picante/picante_{{ version }}.tar.gz
+  sha256: 75e4d73080db67e776562a1d58685438461cbde39af46900c7838da56aef0a62
+
+build:
+  number: 0
+  skip: true  # [win32]
```
Comment (mariusvniekerk) [inline recipes/r-picante/meta.yaml]: You might want to consider just skipping this set on win altogether and do windows compatibility in the generated feedstock

**Comment by nick-youngblut** [comment]
Hey @mariusvniekerk, it doesn't appear that feedstocks were automatically created for `r-caper` or `r-picante`. I thought they would have been created by now (16 hours since the merge). Do I need to do something? Is there a problem due to including 2 recipes in the same PR?

**Pull request merged**